frontend/src/features/projects/common/ProjectSummaryView.tsx

```diff
@@ -39,28 +39,30 @@ const ProjectSummaryView = ({ formikRef }: IStepProps) => {
       >
         {formikProps => (
           <Form>
-            {project.status?.code === ReviewWorkflowStatus.Cancelled && (
+            {(project.status?.code === ReviewWorkflowStatus.Cancelled ||
+              project.status?.code === ReviewWorkflowStatus.TransferredGRE) && (
               <ErpTabs
                 isReadOnly
                 goToGreTransferred={noop}
                 {...{ currentTab, setCurrentTab, setSubmitStatusCode, submitStatusCode }}
               />
             )}
-            {project.status?.code !== ReviewWorkflowStatus.Cancelled && (
-              <>
-                <ReviewProjectForm canEdit={false} />
-                <ProjectNotes disabled={!project.status?.isActive} />
-                <PublicNotes disabled={!project.status?.isActive} />
-                <StepErrorSummary />
-                <StepActions
-                  onSave={() => formikProps.submitForm()}
-                  onNext={noop}
-                  nextDisabled={true}
-                  saveDisabled={!project.status?.isActive}
-                  isFetching={!noFetchingProjectRequests}
-                />
-              </>
-            )}
+            {project.status?.code !== ReviewWorkflowStatus.Cancelled &&
+              project.status?.code !== ReviewWorkflowStatus.TransferredGRE && (
```

Comment:
Adding another case to prevent the duplication of the ReviewProjectForm when displaying the other tabs.

Reply:
I think we have all the statuses covered, but I might be wrong.

Reply:
For now if it solves the main issue I think it'll be okay.  With the changes I'm working I'll try and cover other scenarios.